quiche/src/lib.rs

```diff
@@ -6475,6 +6493,29 @@ impl<F: BufFactory> Connection<F> {
                     qlog_with_type!(QLOG_METRICS, self.qlog, q, {
                         p.recovery.maybe_qlog(q, now);
                     });
+
+                    qlog_with_type!(QLOG_PACKET_LOST, self.qlog, q, {
+                        for epoch in [
+                            packet::Epoch::Initial,
+                            packet::Epoch::Handshake,
+                            packet::Epoch::Application,
+                        ] {
+                            for (header, time_lost) in
+                                p.recovery.get_lost_pkts(epoch)
```

Comment:
One possible worry is that in cases where qlog is enabled but QLOG_PACKET_LOST isn't, get_lost_pkts won't be called, so we'll accumulate a potentially large number of lost packets in this array until `on_pkt_num_space_discarded` is called at a future date.